### PR TITLE
tests: patch webbrowser.open

### LINKED ISSUE #7014: tests: opens browser with plots when running tests
See https://github.com/iterative/dvc/pull/6913#discussion_r754003524.

Can be reproduced with `pytest tests/func/test_live.py`.

### PR DESCRIPTION
**Pull request by daavoo**
Prevent tests from opening browser windows.

Closes #7014

### CODE REVIEW

**Review by pared — APPROVED** [review]

File: tests/func/test_live.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def live_stage(tmp_dir, scm, dvc, mocker):
         pytest.skip("no dvclive")
 
     mocker.patch("dvc.stage.run.Monitor.AWAIT", 0.01)
+    mocker.patch("dvc.repo.live.webbrowser_open")
```
Comment (skshetry) [inline tests/func/test_live.py:69]: I'm worried that this may get unnoticed on new tests. We should mock `webbrowser.open` globally if that's possible.

**Review by skshetry — APPROVED** [review]

File: tests/conftest.py [hunk 2]
```diff
@@ -178,3 +178,9 @@ def custom_template(tmp_dir, dvc):
 
 
 scriptify_fixture = pytest.fixture(lambda: scriptify, name="scriptify")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def webbrowser_open(session_mocker):
+    session_mocker.patch("dvc.repo.live.webbrowser_open")
```
Comment (skshetry) [inline tests/conftest.py:185]: Let's drop this? We may still want to test `webbrowser_open`?

Comment (daavoo) [reply tests/conftest.py:185]: It is still possible to re-patch inside the test and it works as intended

Comment (daavoo) [reply tests/conftest.py:185]: And without it, the global patch doesn't work for the test in `tests/func/live`

Comment (skshetry) [reply tests/conftest.py:185]: I don't think it's a good idea to mock `dvc.repo.live.webbrowser_open` globally.

Comment (skshetry) [reply tests/conftest.py:185]: Merging, let's see if we can fix this after https://github.com/iterative/dvc/pull/6993. We may be able to mock `ui.open_browser`.

Comment (skshetry) [reply tests/conftest.py:185]: Okay, making the imports for `webbrowser` lazy makes it work. Pushing the changes.

**Pull request merged**